src/sc2laddercore/LadderManager.cpp

```diff
@@ -264,30 +264,40 @@ std::string LadderManager::GetBotCommandLine(const BotConfig &AgentConfig, int G
 	{
 	case Python:
 	{
-		OutCmdLine = "python " + AgentConfig.FileName;
+		OutCmdLine = Config->GetValue("PythonBinary") + " " + AgentConfig.FileName;
+		OutCmdLine += " --GamePort " + std::to_string(GamePort) + " --StartPort " + std::to_string(StartPort) + " --LadderServer 127.0.0.1 --OpponentId " + OpponentId;
+		break;
+	}
+	case Wine:
+	{
+		OutCmdLine = "wine \"" + AgentConfig.FileName + "\"";
+		OutCmdLine += " \"--GamePort " + std::to_string(GamePort) + " --StartPort " + std::to_string(StartPort) + " --LadderServer 127.0.0.1 --OpponentId " + OpponentId + "\"";
```

Comment:
>" \"--GamePort "

" --GamePort "
?

Reply:
Ah, I see, you want enclose AgentConfig.FileName into ". However, \" looks wrong. Is it ok?

Reply:
unsure, i think i could even revert that one as a test with a newly compiled 5minbot didnt spit out the error message i got before. I might have been confused by https://github.com/Cryptyc/Sc2LadderServer/pull/67/commits/c6ead412ee01f5809f0fe5688bf0439c9cd766ae